stdlib/Sockets/src/IPAddr.jl

```diff
@@ -251,3 +251,6 @@ struct InetAddr{T<:IPAddr}
 end
 
 InetAddr(ip::IPAddr, port) = InetAddr{typeof(ip)}(ip, port)
+function show(io::IO, addr::InetAddr{T}) where T
+    print(io, "InetAddr{", T, "}(", addr.host, ", ", Int(addr.port), ")")
+end
```

Comment:
This would be better:
```
function show(io::IO, addr::InetAddr)
    show(io, typeof(addr))
    print(io, "(")
    show(io, addr.host)
    print(io, ", ", Int(addr.port), ")")
end
```
We don't want to re-implement how types are printed, and `addr.host` should probably be printed with `show`.